docs/cli.md

```diff
@@ -218,6 +219,19 @@ Usage:
 $ s2i rebuild <image name> [<new-tag-name>]
 ```
 
+# s2i generate
+The `s2i generate` command produces a Dockerfile using an existing S2I base 
+image that can be used to produce an image by any application supporting the
+format.
+
+The *image name* argument is in the URL-like, for example 
+`docker://docker.io/centos/nodejs-10-centos7`.
```

Comment:
In my experience the `docker://` scheme is not used often. I would propose the following:

```
The *image name* is a reference to the builder image that serves as the base of the generated Dockerfile. For example, `docker.io/centos/nodejs-10-centos7:latest`.
```

Reply:
Removing the scheme now would require another PR and would also limit the `containers/image` benefits by limiting the registry type it can fetch metadata from to only Docker (assuming all build images are indeed stored in a Docker registry).

Since this is a Tech Preview, can we move on with this as is for now and discuss how the introduction of OCI registries would look like?

Reply:
@isutton, @sbose78 should we address the documentation bits in #1024?

Reply:
Solved in https://github.com/openshift/source-to-image/pull/1023/commits/fb0be78c7f00a88b37cea07d4fc8e9e6ce59f9e9 and https://github.com/openshift/source-to-image/pull/1023/commits/888560cf0d71289ad9f706773c40c8974847f339